### PR TITLE
workaround command generation problem between upstart & systemd

### PR DESCRIPTION
**Pull request by jorisd**
tested OK on both centos 6 & cnetos 7.

### CODE REVIEW

File: recipes/config.rb [hunk 1]
```diff
@@ -40,9 +40,18 @@
...
+  systemd: {
+    command: "#{bin} #{flags}",
+  },
+}
 
 poise_service 'bamboo' do
-  command bamboo_command
+  node['qubit_bamboo']['poise_service']['options'].each do |k, v|
+    options k, v
+  end if node['qubit_bamboo']['poise_service']['options']
```
Comment (sun77) [inline recipes/config.rb]: OPT : you could remove the if since you declare it just above.

File: recipes/config.rb [hunk 2]
```diff
@@ -40,9 +40,18 @@
 bin = ::File.join(node['qubit_bamboo']['home'], 'bamboo')
 flags = node['qubit_bamboo']['flags'].sort.map { |k, v| " -#{k}=#{v}" }.join ' '
 syslog = node['qubit_bamboo']['syslog'] ? '2>&1 | logger -p user.info -t bamboo' : ''
-bamboo_command = %("#{bin} #{flags} #{syslog}")
+
+node.default['qubit_bamboo']['poise_service']['options'] = {
```
Comment (kamaradclimber) [inline recipes/config.rb:44]: why don't you define this in the attribute file?

Comment (jorisd) [inline recipes/config.rb:44]: because otherwise, it will compute an incomplete command.
If you have a better solution, i'm all ears :)

Comment (pierrecdn) [inline recipes/config.rb:44]: Gregoire said that you can move all the block bin/flags/syslog + the poise_service options hash inside the attributes file.
But it's an OPT for me.

**Comment by pierrecdn** [comment]
OPT: You should add a test in another PR to check that there's at least a non-empty log file at the end of the Chef run.

**Comment by sun77** [comment]
:+1: 
FAR : for me the rspec on the log file is mandatory

File: recipes/config.rb (hunk 1 above)
Comment (jorisd) [inline recipes/config.rb]: done

**Comment by pierrecdn** [comment]
:+1:
